test/js/bun/io/bun-write.test.js

```diff
@@ -290,3 +290,8 @@ it.skip("Bun.write('output.html', HTMLRewriter.transform(Bun.file)))", async don
   expect(await Bun.file(outpath).text()).toBe("<div><blink>it worked!</blink></div>");
   done();
 });
+
+it("#2674", async () => {
```

Comment:
@Jarred-Sumner this will pollute the console **out,** but I added it so we can test on CI/CD at once before merging

Reply:
Can we run this in a separate script that is spawnSync’d?

Reply:
Then we could either redirect to a file or read the buffer in memory in the pipe or both

Reply:
sounds good I will do that

Reply:
done in done in https://github.com/oven-sh/bun/pull/2850/commits/0dcd90d693e4be86eacc373d4d7ecdd1fbfef2cc